media-video/mpv/mpv-0.11.0-r1.ebuild

```diff
@@ -0,0 +1,278 @@
+fi
+DOCS+=( README.md etc/example.conf etc/input.conf )
+
+# See Copyright in source tarball and bug #506946. Waf is BSD, libmpv is ISC.
+LICENSE="GPL-2+ BSD ISC"
+SLOT="0"
+# Here 'opengl' stands for GLX, 'egl' stands for any EGL-based output
+IUSE="+alsa archive bluray cdda +cli doc drm dvb +dvd egl +enca encode +iconv
+jack jpeg lcms +libass libav libcaca libguess libmpv lua luajit openal +opengl
+oss pulseaudio pvr raspberry-pi rubberband samba sdl selinux test v4l vaapi
+vdpau vf-dlopen wayland +X xinerama +xscreensaver xv"
+
+REQUIRED_USE="
+	|| ( cli libmpv )
+	egl? ( || ( X wayland ) )
+	enca? ( iconv )
+	lcms? ( || ( opengl egl ) )
+	libguess? ( iconv )
+	luajit? ( lua )
+	opengl? ( X )
+	pvr? ( v4l )
+	v4l? ( || ( alsa oss ) )
+	vaapi? ( X )
+	vdpau? ( X )
+	wayland? ( egl )
+	xinerama? ( X )
+	xscreensaver? ( X )
+	xv? ( X )
+"
+
+CDEPEND="
+	!libav? ( >=media-video/ffmpeg-2.4.0:0=[encode?,threads,vaapi?,vdpau?] )
+	libav? ( >=media-video/libav-11:0=[encode?,threads,vaapi?,vdpau?] )
+	sys-libs/zlib
+	alsa? ( >=media-libs/alsa-lib-1.0.18 )
+	archive? ( >=app-arch/libarchive-3.0.0:= )
+	bluray? ( >=media-libs/libbluray-0.3.0 )
+	cdda? ( dev-libs/libcdio-paranoia )
+	drm? ( x11-libs/libdrm )
+	dvb? ( virtual/linuxtv-dvb-headers )
+	dvd? (
+		>=media-libs/libdvdnav-4.2.0
+		>=media-libs/libdvdread-4.1.0
+	)
+	egl? ( media-libs/mesa[egl,wayland(-)?] )
+	enca? ( app-i18n/enca )
+	iconv? ( virtual/libiconv )
+	jack? ( media-sound/jack-audio-connection-kit )
+	jpeg? ( virtual/jpeg:0 )
+	lcms? ( >=media-libs/lcms-2.6:2 )
+	libass? (
+		>=media-libs/libass-0.12.1:=[fontconfig,harfbuzz]
+		virtual/ttf-fonts
+	)
+	libcaca? ( >=media-libs/libcaca-0.99_beta18 )
+	libguess? ( >=app-i18n/libguess-1.0 )
+	lua? (
+		!luajit? ( || ( =dev-lang/lua-5.1*:= =dev-lang/lua-5.2*:= ) )
+		luajit? ( dev-lang/luajit:2 )
+	)
+	openal? ( >=media-libs/openal-1.13 )
+	pulseaudio? ( media-sound/pulseaudio )
+	rubberband? ( >=media-libs/rubberband-1.8.0 )
+	samba? ( net-fs/samba )
+	sdl? ( media-libs/libsdl2[sound,threads,video,X?,wayland?] )
+	v4l? ( media-libs/libv4l )
+	wayland? (
+		>=dev-libs/wayland-1.6.0
+		>=x11-libs/libxkbcommon-0.3.0
+	)
+	X? (
+		x11-libs/libX11
+		x11-libs/libXext
+		>=x11-libs/libXrandr-1.2.0
+		opengl? (
+			x11-libs/libXdamage
+			virtual/opengl
+		)
+		vaapi? ( >=x11-libs/libva-1.2.0[X] )
+		vdpau? ( >=x11-libs/libvdpau-0.2 )
+		xinerama? ( x11-libs/libXinerama )
+		xscreensaver? ( x11-libs/libXScrnSaver )
+		xv? ( x11-libs/libXv )
+	)
+"
+DEPEND="${CDEPEND}
+	${PYTHON_DEPS}
+	>=dev-lang/perl-5.8
+	dev-python/docutils
+	virtual/pkgconfig
+	doc? ( dev-python/rst2pdf )
+	test? ( >=dev-util/cmocka-1.0.0 )
+"
+RDEPEND="${CDEPEND}
+	selinux? ( sec-policy/selinux-mplayer )
+"
+
+pkg_pretend() {
+	if ! use libass; then
+		ewarn "You have disabled the libass flag. No OSD or subtitles will be displayed."
+	fi
+
+	if use openal; then
+		ewarn "You have enabled the openal audio output, which is a fallback"
+		ewarn "and disabled by upstream."
```

Comment:
I don't exactly know the original meaning, because it was there before me. But I think it is that `this is a second class feature, avoid it if possible`.
I'll think of something that fits better here.